book/from_flat_php_to_symfony2.rst

```diff
@@ -683,7 +715,7 @@ Better Templates
 If you choose to use it, Symfony comes standard with a templating engine
 called `Twig`_ that makes templates faster to write and easier to read.
 It means that the sample application could contain even less code! Take,
-for example, the list template written in Twig:
+for example, lets rewrite ``layout.html.php`` template in Twig:
```

Comment:
we never use ``lets`` in the doc AFAIK (and it should be ``let's`` anyway if you use it)